qt/qt_common/map_widget.hpp

```diff
@@ -107,6 +116,12 @@ public slots:
   std::unique_ptr<QOpenGLShaderProgram> m_program;
   std::unique_ptr<QOpenGLVertexArrayObject> m_vao;
   std::unique_ptr<QOpenGLBuffer> m_vbo;
+
+  // Fraction of the viewport for a move event
+  static constexpr float VIEWPORT_FRACTION_MOVE = 0.2;
```

Comment:
1. Code style: https://github.com/organicmaps/organicmaps/blob/master/docs/CPP_STYLE.md

E.g. kViewportFractionMove

2. These constants are not used outside of the class, there's no sense in storing them in a header. Using static vars or anonymous namespace in .cpp is enough (if they're used in more than one function).

3. Maybe use Rough and Smooth instead of Light?

Reply:
I think the last push should address those comments.
Thanks!